constraint/pkg/client/drivers/remote/httpclient.go

```diff
@@ -1,289 +0,0 @@
-// Copyright 2017 The OPA Authors.  All rights reserved.
-// Use of this source code is governed by an Apache2
-// license that can be found in the LICENSE file.
-
-package remote
```

Comment:
so are we writing off the `remote` driver here? or does it not make sense anymore? is it an unsupported case now?

Reply:
It has been dead code for a long time. Removing it.